porcupine/plugins/filemanager.py

```diff
@@ -360,7 +363,7 @@ def new_file_here(path: Path) -> None:
     # Doing something to an entire project is more difficult than you would think.
     # For example, if the project is renamed, venv locations don't update.
     # TODO: update venv locations when the venv is renamed
-    Command("New file", "<<FileManager:New file>>", (lambda p: p.is_dir()), new_file_here),
+    Command("New file here", "<<FileManager:New file>>", None, new_file_here),
```

Comment:
This way you don't need to add `None` checks everywhere else:

```suggestion
    Command("New file here", "<<FileManager:New file>>", (lambda p: True), new_file_here),
```